src/Toml/Bi/Map.hs

```diff
@@ -460,6 +462,29 @@ _LByteStringText = prism (TL.encodeUtf8 . TL.fromStrict) eitherText
 _LByteString :: TomlBiMap BL.ByteString AnyValue
 _LByteString = _LByteStringText >>> _Text
 
+-- | Helper bimap between a Word8 lists and Int lists
+_Word8ArrIntArr :: TomlBiMap [Word8] [Int]
+_Word8ArrIntArr = BiMap
+    { forward  = Right . map fromIntegral
+    , backward = eitherNonNeg
+    }
+  where
+    eitherNonNeg :: [Int] -> Either TomlBiMapError [Word8]
+    eitherNonNeg xs
+      | all (>= 0) xs = Right $ map fromIntegral xs
+      | otherwise     = Left $
+          ArbitraryError ("Negative numbers not allowed when converting "
+              <> tShow xs
+              <> " to a ByteString")
+
+-- | Helper bimap for a 'Integer' list and strict 'ByteString'
+_ByteStringArray :: TomlBiMap ByteString [Int]
+_ByteStringArray = iso unpack pack >>> _Word8ArrIntArr
+
+-- | Helper bimap for a 'Integer' list and lazy 'ByteString'
+_LByteStringArray :: TomlBiMap BL.ByteString [Int]
+_LByteStringArray = iso BL.unpack BL.pack >>> _Word8ArrIntArr
```

Comment:
The current implementation of `_Word8ArrIntArr` is okay :+1: But I propose to do it differentl:

1. Implement `_Word8` bidirectional map.
2. Create ByteString array mapping by composing current `iso` and `_Array _Word8`

This solution has the following benefits:

1. It reuses more existing code
2. In implements new primitive codec which can be useful for somebody in the future

Reply:
That's a much cleaner approach. Thanks!